doc/content/resources/multiapps_chps/chp_3_applications.md

```diff
@@ -0,0 +1,122 @@
+- The individual applications can solve on their own space/time scale and with optimal preconditioners and solver settings.
+
+!media media/resources/typical_reactor_multiapps.png
+       style=display: block;margin-left:auto;margin-right:auto;width:80%;
+       id=typical_multiapps
+       caption=A typical hierarchy of MultiApps for nuclear reactor simulation.
+
+[typical_multiapps] illustrates a typical multiphysics approach for reactor modeling. The application with largest spatial domain and longest timesteps, e.g. neutronics (Griffin), is typically selected as the parent application. Griffin typically provides the power distribution ($\mathbb{P}$) to a child application that solves thermal physics and provides the temperature distribution ($\mathbb{T}$) and sometimes thermal expansion as feedback ($e$). For solid-fueled reactors, the thermal physics is typically handled by MOOSE's heat conduction module (or MOOSE apps equipped by it). For liquid-fueled reactors, the thermal physics is usually handled by MOOSE's Navier Stokes module or specific apps such as Pronghorn and NekRS. While the thermal expansion of liquid can be solved by NS equations, thermal expansion of the solid needs to be handled by MOOSE's Tensor Mechanics module. Thermal physics and thermal stress induced by thermal expansion are closely interconnected in solids (known as thermomechanics) and can usually be fully coupled without using MultiApps. It is also feasible to set up tensor mechanics as a grandchild app, which receives temperature from the thermal child application and gives displacement as feedback.
+
+Localized physics such as performance of coolant channels, heat pipes, and fuel can also be handled by grandchild applications. These local applications receive temperature and neutronics information from parent and child applications and perform their specific computation in their specific domain. In some cases, the local physics results are important to reactor thermal and neutronics, such as fuel volumetric swelling strain and coolant temperature, which will need to be transferred back to the parent and child applications for Picard iteration. In other cases, the local physics results do not affect reactor thermal and neutronics physics, such as fuel-cladding interaction, and can be loosely coupled without iteration.
+
+### MultiApp Reactor Examples on VTB
+
+The examples available on the Virtual Test Bed site adopt a number of different coupling approaches based on the specific reactor type and phenomena of focused interest. We briefly summarize some examples to show readers the flexibility in using MultiApps (see [vtb_examples]).
+
+!media media/resources/vtb_examples.png
+      style=display: block;margin-left:auto;margin-right:auto;width:100%;
+      id=vtb_examples
+      caption=Hierarchies of MultiApps for VTB examples.
+
+#### VTB-MSFR
```

Comment:
for msfr, htgr, sfr and pb-fhr we have more than one model.
- let's link to the model in particular rather than the index page
- let's be more precise when referring to the model. For example instead of MSFR example, say MSFR core multiphysics simulation